kamailio/assets/service_checks.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "agent_version": "7.0.0",
```

Comment:
```suggestion
    "agent_version": "7.40.1",
```

Reply:
resolved in cf85a7a2cd46cc4e6c399e639ff477102a24aead